### PR TITLE
Added support for the FEZ CerBot

### PR DESCRIPTION
**Pull request by dhylands**
For this platform: http://www.watterott.com/en/FEZ-Cerbot-Assembled

### CODE REVIEW

File: stmhal/boards/PYBV3/mpconfigboard.h [hunk 1]
```diff
@@ -15,6 +16,7 @@
 #define MICROPY_HW_ENABLE_DAC       (1)
 #define MICROPY_HW_ENABLE_I2C1      (1)
 #define MICROPY_HW_ENABLE_SPI1      (1)
+#define MICROPY_HW_HAS_LED          (1)
```
Comment (dpgeorge) [inline stmhal/boards/PYBV3/mpconfigboard.h]: Probably should be MICROPY_HW_HAS_SPI2.

File: stmhal/main.c [hunk 2]
```diff
@@ -68,6 +68,7 @@ static FATFS fatfs1;
 #endif
 
 void flash_error(int n) {
+#if MICROPY_HW_HAS_LED
```
Comment (dpgeorge) [inline stmhal/main.c]: Since you added dummy led_ functions, you don't need these guards.

**Comment by dpgeorge** [comment]
These 2 boards are sufficiently different that they need 2 separate configs?

File: stmhal/boards/PYBV3/mpconfigboard.h (hunk 1 above)
Comment (dhylands) [inline stmhal/boards/PYBV3/mpconfigboard.h]: Both variants of the SPI2_MISO pins are used for hard-coded functions (the onboard line detectors), so SPI2 can't be used. - Oh yeah - copy/paste error - I see it.

File: stmhal/main.c (hunk 2 above)
Comment (dhylands) [inline stmhal/main.c]: ok - I went back and forth a few times. I thought about just addng an extra define for LED1 and using that, but then having the HW_HAS_LED seemed more consistent.

So I'd like to get some feedback from you about how you want to deal with it.

I see 3 ways:
1 - Use MICROPY_HW_HAS_LED
2 - Use MICROPY_HW_LED1

With either of those, we can choose to use the dummy functions, or not.

**Comment by dhylands** [comment]
The Cerb40 is basically a bare STM32F405 with a 12MHz oscillator. No peripherals of any kind.

The CerBot has 1 LED, and several other on-board peripherals (piezo, line detectors, HBRidge) . So they're definitely different boards.

About the only thing in common between the 2 boards is that they both run a 12 MHz crystal, an they both use a 405 processor.

**Comment by dhylands** [comment]
It looks like the CerBOt has been discontinued by ghi and theere are only a few left in stock at the link I got mine from. So I don't mind creating a repository just for the CerBot board definition (not the whole tree, just the board definition)

**Comment by dpgeorge** [comment]
There hasn't been much demand for support for this board, and it's discontinued, so I'd say we don't put this here.

**Comment by dhylands** [comment]
I'm happy with that. I'll close this and create a repo for it.

**Pull request closed without merging**